src/biz/user/user_init.rs

```diff
@@ -130,14 +147,23 @@ async fn create_workspace_database_collab(
   object_id: &str,
   storage: &Arc<CollabAccessControlStorage>,
   txn: &mut Transaction<'_, sqlx::Postgres>,
+  initial_database_records: Vec<(String, String)>,
 ) -> Result<(), AppError> {
   let collab_type = CollabType::WorkspaceDatabase;
   let mut collab = Collab::new_with_origin(CollabOrigin::Empty, object_id, vec![], false);
   {
     let mut txn = collab.context.transact_mut();
-    let _ = collab
+    let workspace_databases = collab
       .data
       .get_or_init::<_, ArrayRef>(&mut txn, WORKSPACE_DATABASES);
+    // insert the initial database records
+    for (object_id, database_id) in initial_database_records {
+      let map_ref = workspace_databases.push_back(&mut txn, MapPrelim::default());
+      // TODO(Lucas): use the const key here.
```

Comment:
These values are defined in the collab-database. Should we import the collab-database as a dependency here?